rdr_service/api/message_broker_cloud_tasks_api.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+from flask import request
+from flask_restful import Resource
+from dateutil.parser import parse
+
+from rdr_service.app_util import task_auth_required
+from rdr_service.api.cloud_tasks_api import log_task_headers
+from rdr_service.dao.base_dao import BaseDao
+from rdr_service.model.message_broker import MessageBrokerEventData
+
+
+class StoreMessageBrokerEventDataTaskApi(Resource):
+    """
+    Cloud Task endpoint: store message broker event data.
+    """
+    @task_auth_required
+    def post(self):
+        log_task_headers()
+        dao = BaseDao(MessageBrokerEventData)
```

Comment:
Should we just create a `MessageBrokerEventDataDao` instead of accessing the model thru the `BaseDao` since we will need a `dao` anyway for querying later?

Reply:
Nvm looks like we will be querying the `MessageBrokerDao` based on the super call.